### DISCUSSION TITLE
Course Work Discussion

### DISCUSSION THREAD

**Discussion by psquarcetta** [FearlessSolutions/engineering-practice-domains#3] [General]
Hi everyone! I'm currently working on the following Java course (https://www.udemy.com/course/java-the-complete-java-developer-course/?couponCode=ST15MT31224). What has everyone been working on recently?

**Comment by lsamuels-fearless** [comment]
My project is moving to containers on EKS so I'm starting with the LinkedIn Learning course "Learning Kubernetes"
